### PR TITLE
admin: add --help option

### PR DESCRIPTION
**Pull request by uristdwarf**

### CODE REVIEW

File: src/admin/main.cc
Comment (lgsilva3087) [inline src/admin/main.cc:1]: Even if the --help is now accepted. The help output is strange:
image
--help should be a valid command and the exception message should not be printed.

**Review by lgsilva3087 — CHANGES_REQUESTED** [review]
Please see my comment.

File: src/admin/main.cc
Comment (uristdwarf) [reply src/admin/main.cc:1]: I was working on trying to fix up the option handling and got this error too. I thought just keeping a simple one line change would fix it. Guess not.

I'll try fixing it, but I might just decide to refactor the command-line option handling.

Comment (uristdwarf) [reply src/admin/main.cc:1]: From my git stash, I tried fixing it but I decided to keep the change simple now.
image

Comment (lgsilva3087) [reply src/admin/main.cc:1]: I understand, please create the tasks to fix it later.

**Review by lgsilva3087 — APPROVED** [review]
Ready.

**Pull request merged**